src/app/features/vendor/models/vendor.model.ts

```diff
@@ -28,3 +28,14 @@ export interface IDropDownFields {
   name: string;
 }
 
+export interface IVendorData{
```

Comment:
IVendorData and IVendorCreation interfaces are almost the same. Create an interface IVendorBasicInfo that they both extend. There is no need to export this interface if it is not required.

Reply:
ok